README.md

```diff
@@ -97,7 +97,82 @@ Once the development environment is set up, you may proceed to install the requi
    yarn start:stats
    ```
 
-Visit http://localhost:3000/ to see the Webb Stats UI! 🕸️ 🚀
+Visit http://localhost:4000/ to see the Webb Stats UI! 🕸️ 🚀
+
+## Docker
+
+### Setup Container
+
+* Install and run [Docker](https://www.docker.com/)
+* Run Substrate front-end from a Docker container and follow the terminal log instructions.
+```bash
+./docker-dev.sh
+```
+
```

Comment:
Can you add the `.env.` file preparation step, please?

Reply:
i've updated the readme [here](https://github.com/webb-tools/webb-dapp/pull/848/commits/c1abc8a3b4574e07cf06d2c7e34b91370b27bd16#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R135)